app/views/form/check_answers.html.erb

```diff
@@ -1,7 +1,10 @@
 <%= turbo_frame_tag "case_log_form", target: "_top" do %>
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-three-quarters-from-desktop">
-      <h1 class="govuk-heading-l">Check the answers you gave for <%= subsection.id.humanize(capitalize: false) %></h1>
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l"><%= subsection.id.humanize %></span>
+        Check your answers
```

Comment:
Are these changes Dan has asked for? Think that wording was in the original prototype because it was asked for as part of the content review Stu did?

Reply:
Also to note: I had the before and after images for check your answers the wrong way round, if that’s why you asked this question. Images are now showing in the correct order ☺️